helm_deploy/values-dev.yaml

```diff
@@ -82,6 +82,20 @@ generic-service:
     USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_ENABLED: false
     USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_PRIORITY: 0
     USER-ALLOCATIONS_RULES_ESAP-ASSESSMENTS_ALLOCATE-TO-USER: # TODO: Determine appropriate user
+    USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_ENABLED: false
+    USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_PRIORITY: 0
+    USER-ALLOCATIONS_RULES_EMERGENCY-ASSESSMENTS_ALLOCATE-TO-USERS_EAST-MIDLANDS: # TODO: Determine appropriate user
```

Comment:
I'm a little bit wary of making emergency allocations a matter of configuration. I think we should lean on the `UserQualification.EMERGENCY` role, as any changes to who would be the emergency assessor would mean a code change.

Reply:
I've updated the user allocator to support allocating by qualification and changed the rule to allocate using the `EMERGENCY` qualification.